packages/rendermime-interfaces/src/index.ts

```diff
@@ -227,6 +228,20 @@ export namespace IRenderMime {
     readonly default: IExtension | ReadonlyArray<IExtension>;
   }
 
+  /**
+   * The IJLIcon interface, which supplies element, render,
+   * and unrender functions
```

Comment:
I think we should mention here that there is an implementation and some example renderers in `@jupyterlab/ui-components`.

Reply:
Thanks! I was hoping to see an example of how it is used.